src/docs/backup.dox

```diff
@@ -240,6 +240,16 @@ An example of opening the backup data source for log-based incremental backup:
 
 @snippet ex_all.c incremental backup
 
+@section backup_export Export tables using backup cursor
+
+The URI \c backup:export can be used to generate \c WiredTiger.export - a text file that
+contains metadata for all objects in the database. The file can be specified as the value
+for \c metadata_file to import a table. See WT_SESSION::create for more details.
+The cursor operates like a normal backup cursor, it can be used to iterate over all files mentioned
+in \c file: metadata entries. The main difference is that ::wiredtiger_open will
```

Comment:
@sueloverso thank you for the comment. Initially, I was thinking to put it that way. But then I realized that the cursor also returns `WiredTigerHS.wt` which is not needed for import (backup). 
Also, I think it's OK, but I want to confirm with you one thing. The cursor doesn't return any `object:` entries, even though they are present locally after `flush_tier` call. I think we don't have to worry about them because they are already in the shared store and in order to import them the user does not have to copy them locally. Am I getting it right?

Reply:
For now actual backup of a tiered database is not done. There is a design for it but it is not part of this work.

Reply:
Thank you! Done.